enhydris_synoptic/templates/enhydris-synoptic/groupstation-report.html

```diff
@@ -3,23 +3,25 @@
 
 {% if object.error %}
   {% blocktrans %}
-    <p>Something's wrong with the data of this report. Check if the
-      time series are all the same time step.</p>
+    <div class="alert alert-warning" role="alert">Something's wrong with the data of this report. Check if the time series are all the same time step.</div>
   {% endblocktrans %}
 {% else %}
   {% localtime off %}
-    <dl class="dl-horizontal">
-      <dt>{% trans "Last update" %}</dt><dd>{{ object.last_common_date_pretty }}</dd>
-      <dt>&nbsp;</dt><dd></dd>
-      {% for syntsg in object.synoptic_timeseries_groups %}
-        <dt>
-          {{ syntsg.title|default:syntsg.timeseries_group.name }}
-          {% if syntsg.subtitle %}
-            ({{ syntsg.subtitle }})
+    <dl class="row">
+      <dt class="col-sm-2 text-right">{% trans "Last update" %}</dt>
+      <dd class="col-sm-10">{{ object.last_common_date_pretty }}</dd>
+      <dt class="col-sm-2 text-right">&nbsp;</dt>
+      <dd class="col-sm-10"></dd>
+      {% for synts in object.synoptic_timeseries %}
+        <dt class="col-sm-2 text-right">
+          {{ synts.title|default:synts.timeseries.name }}
+          {% if synts.subtitle %}
+            ({{ synts.subtitle }})
           {% endif %}
         </dt>
-        {% with precision=syntsg.timeseries_group.precision|default:0 %}
-          <dd>{{ syntsg.value|floatformat:precision }} {{ syntsg.timeseries_group.unit_of_measurement.symbol }}</dd>
+        {% with precision=synts.timeseries.precision|default:0 %}
+          <dd class="col-sm-10">{{ synts.value|floatformat:precision }} {{ synts.timeseries.unit_of_measurement.symbol
+            }}</dd>
```

Comment:
Apparently this is not allowed in Django. You can't break the line before the `}}`. The whole thing from `{{` to `}}` must be on the same line.